### PR TITLE
Feat/prsd 491 fix UI errors

### PR DESCRIPTION
**Pull request by JasminConterioSW**
Fixes a couple of UI errors from QA of [PRSD-491](https://mhclgdigital.atlassian.net/browse/PRSD-491)

- Updated the button text on the Select Local Authority page
- Updated basicTextInput so that the input can be given a width (note only a few widths are available)
- Made some of the input fields smaller on the Manual Address Form Page

image
image

[Figma link](https://www.figma.com/design/ka4URNl45NiQO0LoY8XNe4/MHCLG---Private-Rented-Sector-Database?node-id=8457-357084&t=fA7ZAxCm1svNK0D9-1)

### CODE REVIEW

**Review by isobel-softwire — APPROVED** [review]
LGTM 👍 - left a comment about a message key

**Review by AEPR — COMMENTED** [review]
Opened a discussion on the approach, happy to have a chat about it. Everything here is good though and happy to approve we decide to stick with just one fragment.

File: src/main/resources/templates/fragments/forms/basicTextInput.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
...
+<th:block th:fragment="textInput(label, fieldName, hint, fieldWidth)"
           th:replace="~{fragments/forms/inputGroup :: inputGroup(${label}, ${fieldName}, ~{::input}, ${hint})}">
-    <input class="govuk-input" th:classappend='${#fields.hasErrors(fieldName) ? "govuk-input--error" : ""}' type="text"
+    <input class="govuk-input"
+           th:with='widthClass=${fieldWidth != null ? " govuk-input--width-"+fieldWidth : ""}'
+           th:classappend='${#fields.hasErrors(fieldName) ? "govuk-input--error" : ""} + ${widthClass}'
+           type="text"
            th:field="*{__${fieldName}__}"
            th:attrappend="aria-describedby=${hint != null} ? ${fieldName}+'-details-hint'">
 </th:block>
```
[the remark this answers is not in the document]
Comment (JasminConterioSW) [reply src/main/resources/templates/fragments/forms/basicTextInput.html:10]: I think we have been trying to stick to only one one fragment per file, so if we did this we'd want to split them into two files (which we can do). We've got a similar approach (having the option to add a width) for the numericalInput so we'd want to do the same there. I think either way works, it doesn't feel like it needs to be a separate fragment to me though!

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -308,6 +308,7 @@ forms.selectLocalAuthority.fieldSetHint=Select the local authority that your pro
 forms.selectLocalAuthority.paragraph.one.beforeLink=You can
 forms.selectLocalAuthority.paragraph.one.linkText=find your local authority by searching using the property postcode (opens in new tab).
 forms.selectLocalAuthority.select.label=Select a local authority
+forms.selectLocalAuthority.submitButton.text=Use this address
```
[the remark this answers is not in the document]
Comment (JasminConterioSW) [reply src/main/resources/messages.properties:311]: Looking again, it should also be this on the Select Address page, both here and for the landlord registration journey...

[the remark this answers is not in the document]
Comment (JasminConterioSW) [reply src/main/resources/messages.properties:311]: So I've updated this on the Select Address pages.
But also thought that "Use this address" maybe doesn't make sense on the Select Local Authority page... I've[ asked Sam and Nicola about it on slack](https://softwire.slack.com/archives/C070WSGTGQY/p1733950610224899) so might be worth seeing what they say before merging!

File: src/main/resources/templates/fragments/forms/basicTextInput.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (AEPR) [reply src/main/resources/templates/fragments/forms/basicTextInput.html:10]: I've added a PR to this one (and merged it in to this branch) and did indeed put them into different files as you correctly say.

File: src/main/resources/messages.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (JasminConterioSW) [reply src/main/resources/messages.properties]: Changed this one back to "Save and continue"!

**Pull request merged**